python/pyspark/pandas/frame.py

```diff
@@ -8659,6 +8664,14 @@ def sample(
         falcon         2          2                 10
         fish           0          0                  8
 
+        A random 50% sample of the ``DataFrame``, while ignoring the index.
+
+        >>> df.sample(frac=0.5, random_state=1, ignore_index=True)  # doctest: +SKIP
```

Comment:
qq: why do we skip this ?

Reply:
The sampled rows may not be fixed even with the seed, so we skip the doctests as well as the unit tests.